app/src/main/cpp/DeviceDelegate.h

```diff
@@ -101,6 +101,13 @@ class DeviceDelegate {
   virtual void OnControllersReady(const std::function<void()>& callback) {
     callback();
   }
+  virtual int64_t GetColorFormat(std::string extension) {
+    if (extension == ".ktx") {
+      return GL_COMPRESSED_RGB8_ETC2;
+    } else {
+      return GL_RGBA8;
```

Comment:
I very much prefer the ternary operator for this use cases:
`return extension == ".ktx" ? GL_COMPRESSED_RGB8_ETC2 : GL_RGBA8;`